homework.md

```diff
@@ -56,31 +56,31 @@ When people want to show off code to each other they can put it on Github
 
 What happens when you `fork` a repository?
 
-You delete it
+You make a copy that stays on your github account
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You make a copy that stays on your computer
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+A request that someone adds the changes that you made to a file
```

Comment:
Pull request is you alerting the owner/owners of a certain repository of the changes that you have made in their repo so that they can review your changes and potentially merge those changes or discard those changes.